docs/coordinate.md

```diff
@@ -0,0 +1,50 @@
+
+Coordinate parses Latitude and Longitude.
+
+You can use the `Coordinate` data type for storing coordinates. Coordinates can be defined using one of the following formats:
+
+1. Tuple format: `(Latitude, Longitude)`. For example: `(41.40338, 2.17403)`.
+2. `Coordinate` instance format: `Coordinate(latitude=Latitude, longitude=Longitude)`. For example: `Coordinate(latitude=41.40338, longitude=2.17403)`.
+3. Shorthand format: `(Latitude)`. In this case, the latitude value is specified, and the longitude is set to 0. For example: `(41.40338)`.
```

Comment:
I think we ended up treating this as an error...

Reply:
Indeed, I will change